src/cloudant/design_document.py

```diff
@@ -48,6 +48,44 @@ def __init__(self, database, document_id=None):
         self.setdefault('indexes', dict())
 
     @property
+    def rewrites(self):
+        """
+        Provides an accessor property to a list of dictionaries with rewrite
+        rules in the locally cached DesignDocument.  Each rule for URL rewriting
+        is a JSON object with four fields: ``from``, ``to``, ``method``,
+        and ``query``.
+
+        Note: Requests that match the rewrite rules must have a URL path that
+        starts with ``/$DATABASE/_design/doc/_rewrite``.
+
+        Rewrite rule example:
+
+        .. code-block:: python
+
+            # Add the rule to ``rewrites`` and save the design document
+            ddoc = DesignDocument(self.db, '_design/ddoc001')
+            ddoc['rewrites'] = [
+                {"from": "/old/topic",
+                 "to": "/new/",
+                 "method": "GET",
+                 "query": {}
+                 }
+            ]
+            ddoc.save()
+
+        Once the rewrite rule is saved to the remote database, the GET
+        request URL ``/$DATABASE/_design/doc/_rewrite/old/topic?k=v`` would be
+        rewritten as ``/$DATABASE/_design/doc/_rewrite/new/k=v``.
```

Comment:
You're correct.  After additional tests, the documentation needs to be updated.  Here are the steps I did and results using `animaldb` sample database:
First, set `rewrites` property: 
```
 {
       "from": "/latin_name/",
       "to": "/new/latin_name",
       "method": "GET",
       "query": {
       }
   }
```
Then, run either curl command: 
`curl http://localhost:5984/animaldb/_design/views101/_rewrite/latin_name/?limit=1`
Or
`curl http://localhost:5984/animaldb/_design/views101/_rewrite/latin_name?limit=1`
Finally, the rewrite result:
`/animaldb/_design/views101/new/latin_name?limit=1`